providers/bitbucket.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
                     let commits = [];
                     for (let j = 0; j < project.commits.length; j++) {
                         let commit = project.commits[j];
-                        let message = (commit.message.length > 50) ? commit.message.substring(0, 47) + "..." : commit.message;
+                        let message = (commit.message.length > 256) ? commit.message.substring(0, 253) + "..." : commit.message;
```

Comment:
What he pasted is a unicode character for ellipsis, `…`. You just posted three `.` in a row. You can tell the difference if you attempt to highlight each